packages/legacy/src/common/constants.ts

```diff
@@ -106,10 +106,10 @@ export enum StepType {
 
 export const dnsLabelNameSchema = yup
   .string()
-  .max(63)
-  .matches(/^(?=.{1,253}$)[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/, {
+  .max(253)
+  .matches(/^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/, {
     message: ({ label }) =>
-      `${label} can only contain lowercase alphanumeric characters, dashes and dots, and must start and end with an alphanumeric character, see k8s documentation for more details.`,
+      `${label} must be a valid Kubernetes name (i.e., must contain no more than 253 characters, consists of lower case alphanumeric characters , '-' or '.' and starts and ends with an alphanumeric character).`,
```

Comment:
oww, didn't notice the , `must contain no more than 253 characters` that is too much information for a help text, more like a "learn more" thing,

I'll remove it in a folowup

Reply:
I'm missing the "for more information see k8s documention"

I'll add it in a folowup

Reply:
This is not added for the [provider name error message.](https://github.com/kubev2v/forklift-console-plugin/blob/98d1719f5782d8d4e94bbf39f4212489f8999b52/packages/forklift-console-plugin/src/modules/Providers/views/create/components/ProviderCreateForm.tsx#L109)  I think we should try being consisted. 

Once we say `a valid k8n entity name`, it's implies that you can check the docs, but anyway it's nice to be added as well.
 If you are convinced,  I'll  add for both the legacy and the providers name something like the following?:  "`For more information, see the Kubernetes API documentation`"